src/main/app/claims/claimModelConverter.ts

```diff
@@ -176,4 +177,13 @@ export class ClaimModelConverter {
     }
     return interestDate
   }
+
+  private static convertPaymentDetails (payment: Payment): Payment {
```

Comment:
It doesn't look like this method is converting anything, seems to be making a shallow clone instead. Could we perhaps rename it to be more accurate?

Reply:
Yeah, that is one of the more dirty parts of that PR. I renamed it to be `makeShallowCopy` and added extra comment explaining why we have to do it this way.